config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,39 @@
+CarrierWave.configure do |config|
+
+  # Use local storage if in development or test
+  if Rails.env.development? || Rails.env.test?
+    CarrierWave.configure do |config|
+      config.storage = :file
+    end
+  end
+
+  # Use AWS storage if in production
+  if Rails.env.production?
+    CarrierWave.configure do |config|
+      config.storage = :fog
+    end
+  end
+
+  config.fog_credentials = {
+    :provider               => 'AWS',                             # required
+    :aws_access_key_id      => 'AKIAIHDXB2LV5GV3UARA',            # required
+    :aws_secret_access_key  => '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',     # required
```

Comment:
Line is too long. [89/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.